pyproject.toml

```diff
@@ -14,24 +12,29 @@ classifiers = [
   "License :: OSI Approved :: MIT License",
   "Natural Language :: English",
   "Operating System :: OS Independent",
-  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3"
 ]
+description = "Normalizing flows in PyTorch"
 dynamic = ["dependencies", "version"]
 keywords = [
-    "torch",
-    "normalizing flows",
-    "probability",
-    "distribution",
-    "generative",
-    "deep learning",
+  "torch",
+  "normalizing flows",
+  "probability",
+  "distribution",
+  "generative",
+  "deep learning"
 ]
+name = "zuko"
 readme = "README.md"
 requires-python = ">=3.8"
 
 [project.optional-dependencies]
 dev = [
+  "pre-commit",
   "pytest",
-  "ruff",
+  "python-lsp-ruff",
+  "python-lsp-server[all]",
```

Comment:
Why are `python-lsp-ruff` and `python-lsp-servel[all]` necessary?

Reply:
I accidentally copy pasted it here and now removed it.